console/billing/how-to/add-billing-contact.mdx

```diff
@@ -13,11 +13,11 @@ categories:
   - billing
 ---
 
-The **billing contact** is an additional contact linked to your account who will receive your monthly invoices. This can be useful if invoices in your Organization are handled by another department. 
+**Billing contact** is an additional contact linked to your account who will receive your monthly invoices. This can be useful if invoices in your Organization are handled by another department.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```

Comment:
I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?

Reply:
yes we have to add the iam req 🙏🏻 
the Iam requirement should be included in all pages except certain IAM pages hehe (and some account pages too)